boltdesignsystem/bolt
# DS-318 add disabled state to typeahead

## Jira

https://pegadigitalit.atlassian.net/browse/DS-318

## Summary

Adds disabled state and related styles to the Typeahead component.

## Details

1. CSS is updated to represent disabled state of the search input.
2. A disabled prop is added for the component.

## How to test

Run the branch locally and test the `disabled` prop for the Typeahead component. Make sure if the prop is set to true, the search input is grayed out (looks disabled) and no controls work (both input and buttons are disabled as well).
